client/my-sites/theme/main.jsx

```diff
@@ -240,7 +241,25 @@ const ThemeSheet = React.createClass( {
 		return <ThemeDownloadCard theme={ this.props.id } href={ this.props.download } />;
 	},
 
-	render() {
+	renderEmptyContent() {
```

Comment:
Maybe calling it `renderError()` would be more semantic? See the logic inside `render()`.